packages/mobile/src/components/core/TextInput.tsx

```diff
@@ -242,6 +243,7 @@ export const TextInput = forwardRef<RNTextInput, TextInputProps>(
             value={value}
             onFocus={handleFocus}
             onBlur={handleBlur}
+            placeholder={label && !isFocused ? undefined : placeholder}
```

Comment:
Good q so for cases when there is a label, that's in the placeholder spot till you click into it, the label animates up, and then a placeholder text can appear. Otherwise there is a collision between the two